content/user/upgrading/release_specific_upgrade_considerations.md

```diff
@@ -10,6 +10,10 @@ This document lists things you may wish to take into account as you upgrade.  Th
 - changes to the database schema or contents which deserve special notice
 - changes to basic template operation 
 
+### Zen Cart 1.5.8
+
+- The function `zen_cfg_read_only` has been added to the core.  This means any plugin authors who use this function should wrap it in `if (!function_exists('zen_cfg_read_only'))`. 
+
```

Comment:
IIRC there's only one plugin that ever did that, right?

Reply:
No there's a few in addition to the sitemap. 

Google_mapinator
Multi-Language-Country-Names
dangling_carrot

And there may be others that are not in the Plugins area.   (Installed modifications, commercial mods, etc.)